srcs/django_code/front/js/main.js

```diff
@@ -5,6 +5,7 @@ import { Minichat } from "../components/minichat/minichat.js";
 import { Message } from "../components/minichat/message.js";
 import { Friend } from "../components/minichat/friend.js";
 import { ChatInput } from "../components/minichat/input.js";
+import { trans } from "./helpers.js";
```

Comment:
not use

Reply:
```suggestion

```

Reply:
fix ici [107a78b](https://github.com/Rreyth/get-transcended/pull/61/commits/107a78b9b4740008433407bdf673aafba22facbf)